content/drone-plugins/drone-gcr/index.md

```diff
@@ -8,6 +8,73 @@ logo: google_gcr.svg
 image: plugins/gcr
 ---
 
-{{% alert error %}}
-We are looking for volunteers to port this plugin to drone 0.5
+{{% alert info %}}
```

Comment:
Not sure if this is a valid use of this.

Reply:
This is not valid anymore since it's managed within drone-plugins/drone-docker now.

Reply:
Scratch that, I was just talking about the deleted sentence.

Reply:
Maybe an informational alert box can point people to the additional options that drone-docker provides?